### ISSUE TITLE
ERROR: Service 'nginx' failed to build : Build failed // yarn.lock ;  git protocol 

### ISSUE THREAD

**Issue by krannnn** [doccano/doccano#1737]
How to reproduce the behaviour
---------
Clone `doccano` from github repository:
```bash
git clone https://github.com/doccano/doccano.git
```

```bash
cd doccano
COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1 
docker-compose -f docker/docker-compose.prod.yml --env-file ./docker/.env.example up
```
```
#11 13.60 done.
#11 13.72 yarn install v1.22.17
#11 13.79 [1/4] Resolving packages...
#11 14.16 [2/4] Fetching packages...
#11 25.17 error Command failed.
#11 25.17 Exit code: 128
#11 25.17 Command: git
#11 25.17 Arguments: ls-remote --tags --heads git://github.com/adobe-webplatform/eve.git
#11 25.17 Directory: /app
#11 25.17 Output:
#11 25.17 fatal: remote error: 
#11 25.17   The unauthenticated git protocol on port 9418 is no longer supported.
#11 25.17 Please see https://github.blog/2021-09-01-improving-git-protocol-security-github/ for more information.
#11 25.17 info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.
------
executor failed running [/bin/sh -c apt-get update  && apt-get install -y git python3 make g++  && yarn install  && yarn build  && apt-get clean]: exit code: 128
ERROR: Service 'nginx' failed to build : Build failed
```
This wasn't a problem a few days ago, I guess it's due to the update on the yarn.lock 2 days ago.
Your Environment
---------

*   Operating System: Ubuntu
*   Python Version Used:
*   When you install doccano: March 15th 2022
*   How did you install doccano (Heroku button etc): docker-compose

**Comment by Hironsan** [comment]
Related to:
- https://github.blog/2021-09-01-improving-git-protocol-security-github/

**Comment by inspmoore** [comment]
+1 Anyone has an idea how to fix that?

**Comment by Hironsan** [comment]
Fixed in #1739
